developer_setup.md

```diff
@@ -156,7 +156,7 @@ git fetch other_user
 ```bash
 bin/setup                  # Installs dependencies, config, prepares database, etc
 bundle exec rake evm:start # Starts the ManageIQ EVM Application in the background
-bundle exec rails s        # Starts the application server
+bundle exec rails s        # Starts the application server - no need to run this if evm:start was executed 
```

Comment:
While clarification here might be needed, saying this exact thing is bit misleading; The default setup happens to include a UI worker but you could have changed that in your configuration. 

Since this _is_ the intro developer setup, perhaps simply removing `rails s` would be less confusing. Knowing how to turn off your UI worker and instead run it yourself in the console is beyond the scope of this very particular set of instructions.